spec/instrumentation_spec.rb

```diff
@@ -13,6 +13,11 @@
   let(:events) { [] }
   let(:subscriber) { lambda { |*args| events << ActiveSupport::Notifications::Event.new(*args) } }
 
+  before :all do
+    require 'active_support'
```

Comment:
@olleolleolle can you please move these requires to the top of the file, right after the `require 'spec_helper'`?